pkg/hyper/hyper.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/golang/glog"
 	"github.com/golang/protobuf/proto"
 
+	"github.com/rajatchopra/ocicni"
```

Comment:
Could you also moves this to a local packages? It will be moved into cri-o, refer [here](https://github.com/kubernetes-incubator/cri-o/pull/295#issuecomment-271026169).